dracut-functions.sh

```diff
@@ -732,16 +732,26 @@ fs_get_option() {
 check_kernel_config() {
     local _config_opt="$1"
     local _config_file
-    [[ -f $dracutsysrootdir/boot/config-$kernel ]] \
```

Comment:
I think here we should add a check:
```
[[ $no_kernel == yes ]] && return 0
```

If `--no-kernel` is specified then `$kernel` will be the default value of `uname -r`, so these checks would then run against the current kernel version which does not make sense.

Reply:
Looks like this is an already existing issue